### PR TITLE
Translate the blog Fine-tuning LLMs to 1.58bit: extreme quantization made easy into Chinese

### PR DESCRIPTION
**Pull request by ZeusXuan**
Add the file zh/1_58_llm_extreme_quantization.md and modify the correspondding zh/_blog.yml

### CODE REVIEW

**Comment by pcuenca** [comment]
cc @adeenayakup

File: zh/_blog.yml [hunk 1]
```diff
@@ -1973,3 +1973,13 @@
   tags:
     - research
     - community
+
+- local: 1_58_llm_extrem_quantziation
```
Comment (xianbaoqian) [inline zh/_blog.yml]: typo: 1_58_llm_extreme_quantization

**Review by xianbaoqian — APPROVED** [review]

**Comment by xianbaoqian** [comment]
Thanks for contributing the Chinese version! Fixed some nits.

**Comment by xianbaoqian** [comment]
@ZeusXuan you're very welcome to join our translators group if you're interested! Let us know : )

**Comment by ZeusXuan** [comment]
@xianbaoqian Thank you! I am delighted to be part of the translators group and to contribute to the HuggingFace community during my free time at school and work.

**Comment by xianbaoqian** [comment]
@ZeusXuan ^^ Feel free to add me on wechat. Mine is zhou_a_zhou

**Pull request merged**